addons/website/views/snippets/s_framed_numbers.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_framed_numbers" name="Framed numbers">
+    <section class="s_framed_numbers pt80 pb80">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12">
+                    <h2 style="text-align: center;">Insights, stats, and metrics</h2>
+                    <p class="lead" style="text-align: center;">This section allows to highlight key statistics.<br/><br/></p>
+                </div>
+            </div>
+            <div class="row o_cc o_cc1">
+                <div class="col-lg-4 o_cc o_cc2 pt48 pb48 text-center">
+                    <span class="display-1">12k</span>
+                    <p class="fw-bold">Useful options</p>
+                </div>
+                <div class="col-lg-4 o_cc o_cc2 pt48 pb48 text-center">
```

Comment:
Because of `text-center` the heading cannot be aligned left or right.
<img width="394" alt="image" src="https://github.com/user-attachments/assets/7e2e1617-c144-4f29-ad89-216a3d6d36de">
Could you test appling `text-center` to the heading and paragraph directly instead?